kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/FilterService.java

```diff
@@ -935,6 +950,15 @@ public void handleQuickSearchDropdownByBucket(String value,boolean coded) {
 
             clearFilters(searchBE);
 
+            // add definitions
+            List<String> definitions = getListDefinitionCodes(code);
+
+            for(int i=0;i< definitions.size(); i++){
+                if(i== 0) {
+                    searchBE.add(new Filter(Attribute.LNK_DEF, Operator.STARTS_WITH, definitions.get(i)));
```

Comment:
Should this be `Operator.CONTAINS`?

Reply:
The array would break if LNK_DEF has arrays by CONTAINS.